couchbase-server/Dockerfile

```diff
@@ -50,9 +50,12 @@ RUN groupadd -g1000 couchbase && \
 
 # Install couchbase
 RUN rpm --install $CB_RELEASE_URL/$PROD_VERSION/$CB_PACKAGE
+RUN chmod -R o+rw /opt/couchbase/etc
```

Comment:
This seems to be a strange setting, as I believe /opt/couchbase/etc is a directory?  Did it already have execute permissions for 'other'?

Reply:
We don't need to create file in that directory, just read and write to existing ones. I found out recently that ns_serer writes 99% of it's config to /opt/couchbase/var, but writes a small number of things (like data paths) to /opt/couchbase/etc.

Reply:
While technically one can read/write to files within the directory, not being able to go into (cd) the directory seems... awkward.  Is there any reason to NOT give '+x' for it?

Reply:
I that I can think of. I didn't add it since I figured the less permissions I gave to other users to better. I can change this if you would like.

Reply:
Please do update it, it would seem less strange for me. :)